tests/test_replace_aliases.py

```diff
@@ -1,6 +1,10 @@
 # coding: utf-8
-from __future__ import (unicode_literals, print_function,
-                        absolute_import, division)
+from __future__ import (
+    unicode_literals,
+    print_function,
+    absolute_import,
+    division,
+)
```

Comment:
Ok, please remove all the `__future__` import I forgot. I'm really sorry about that.